src/ingredients.ts

```diff
@@ -16,9 +16,9 @@ const createSpanBtns = () => {
 }
 
 export const createIngredientsListFromLocalStorage = () => {
-    const receivedIngredients = Storage.get('ingredients')
+    const receivedIngredients: Array<string> = Storage.get('ingredients')
 
-    receivedIngredients.forEach(ingredient => {
+    receivedIngredients.forEach((ingredient: string) => {
```

Comment:
not necessary, TypeScript will know it's a string as you typed `receivedIngredients`